src/endpoints/esdt/esdt.service.ts

```diff
@@ -235,6 +252,22 @@ export class EsdtService {
       delete tokenProps.NFTCreateStopped;
     }
 
+    switch (elasticProperties.type) {
```

Comment:
duplicated code with L173-L187. Might extract a function in a helper package or something

Reply:
done, thanks !